src/main/java/com/kiryometsy/applicator/job/simple/controllers/BestOffersController.java

```diff
@@ -0,0 +1,31 @@
+package com.kiryometsy.applicator.job.simple.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.reactive.function.client.WebClient;
+
+@RestController
+@RequestMapping("/best-offers")
+public class BestOffersController {
+
+    @Autowired
+    private WebClient.Builder webClientBuilder;
+
+    @Value("${host.name}")
+    private  String hostName;
+
+
+    @GetMapping
+    public String namedParams(@RequestParam(value="page", required = false) String page)
+    {
+        // backend?criteria=seniority%3Dtrainee,junior%20requirement%3Djava&page=1
+
+        webClientBuilder.build()
+                .get()
+                .uri(hostName + "backend?criteria=seniority%3Dtrainee,junior%20requirement%3Djava&page=1")
+                .retrieve();
+        return hostName + "backend?criteria=seniority%3Dtrainee,junior%20requirement%3Djava&page=1";
```

Comment:
You've exposed a possible problem which I missed during task creation. Let's fix it. As you see, there can be more than one page of results.
There can be several ways to solve it:
1. Easy path. Play with parameters. Though, NFJ doesn't disclose their API documentation, you can try not to pass page parameter, or add, e.g. 'limit=1000' parameter to retrieve all the results in one call.
2. Hard path. If the p.1 won't work, you have to retrieve pages one by one, until you receive HTTP error or empty results.
2.1. Check out NFJ's API responses thoroughly. Usually, there's somewhere a number with overall search results in JSON. You can use this number to know in advance how many calls you have to do.